### PR TITLE
Initial Django app creation

### LINKED ISSUE #6: Create a hello world
Need the directory structure in place along with all necessary Django and Angular libraries in the repo.

### PR DESCRIPTION
**Pull request by randyxli**
Resolves #6

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+# Rope
+.ropeproject
+
+# Django stuff:
+*.log
+*.pot
+
+# Sphinx documentation
+docs/_build/
```
Comment (jaypatel11) [inline .gitignore:51]: Add .sqlite3

**Comment by jaypatel11** [comment]
+1, good enough for init directory structure

**Comment by jaypatel11** [comment]
@cjunmokim Any thoughts here?

**Comment by cjunmokim** [comment]
Looks good! Can merge

**Pull request merged**